pages/give-devices.js

```diff
@@ -108,10 +108,10 @@ export default function GiveDevicesPage() {
       </Head>
       <Navbar />
       <main>
-        <div className="m-auto w-3/4 p-10 my-10 bg-green-300 border border-gray-300">
-          <div className="mt-10 sm:mt-0">
+        <div className="p-12">
```

Comment:
Might be worth changing this to `sm:p-12` the reason being that on smaller devices (such as phones) this padding can be quite significant as it squashes the text together. Having no padding on these devices appears to work quite well and makes good use of the extra space given to it.

Noting: The updating of the CSS breakpoints does not appear to work properly on the Vercel host for me currently, however this should provide improved responsiveness.